app/views/jobseekers/job_applications/review.html.slim

```diff
@@ -1,13 +1,13 @@
 - content_for :page_title_prefix, t(".title")
 
-= render(Jobseekers::JobApplicationHeadingComponent.new(vacancy: @job_application.vacancy))
+= render(Jobseekers::JobApplicationHeadingComponent.new(vacancy: job_application.vacancy, back_path: jobseekers_job_application_build_path(job_application, :declarations)))
 
 .govuk-grid-row
   .govuk-grid-column-two-thirds
-    .govuk-caption-m Step 8 of 8
+    .govuk-caption-m Step 10 of 10
```

Comment:
Not for this PR, but I reckon we should rethink our step numbering entirely at some point so we don't need to hardcode this!

Reply:
I envisaged it happening as part of implementing the process steps sidebar. We have something similar for publishers, but I can't remember if that got refactored as well yet or not?